src/VisualStudio/Core/Def/Interactive/ScriptingOleCommandTarget.cs

```diff
@@ -29,10 +29,7 @@ internal sealed class ScriptingOleCommandTarget : AbstractOleCommandTarget
         {
             var result = WpfTextView.GetBufferContainingCaret(contentType: ContentTypeNames.RoslynContentType);
 
-            if (result == null)
-            {
-                result = WpfTextView.GetBufferContainingCaret(contentType: PredefinedInteractiveCommandsContentTypes.InteractiveCommandContentTypeName);
-            }
+            result ??= WpfTextView.GetBufferContainingCaret(contentType: PredefinedInteractiveCommandsContentTypes.InteractiveCommandContentTypeName);
```

Comment:
For this scenario I would prefer something like

```csharp
return WpfTextView.GetBufferContainingCaret(contentType: ContentTypeNames.RoslynContentType)
    ?? WpfTextView.GetBufferContainingCaret(contentType: PredefinedInteractiveCommandsContentTypes.InteractiveCommandContentTypeName);
```

Reply:
yup!  that seems like a nice additional improvement we could do in this space.  e.g. "prefer coalesce vs multiple assignments and tests"